RestApi/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,3 @@
+using System;
+
+[assembly: CLSCompliant(false)]
```

Comment:
:memo: This addresses warnings in the generated code and also speeds up the build. There are not really any downsides for C# code.